src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String FILE_NOT_EXIST = "src/main/resourses/input_fake_file.csv";
+    private static final String FILE_READ_FOR_TEST = "src/test/resourses/file_read_test.csv";
+    private static final String FILE_WRITE_FOR_TEST = "src/test/resourses/file_write_test.csv";
+    private FileReader fileReaderTest;
+    private FileWriter fileWriterTest;
```

Comment:
if you test writer you shouldn't depend on reader
same goes for every other test class